src/cli.ts

```diff
@@ -107,6 +107,17 @@ const build = debounce(async function build() {
     }
     let warned = false;
 
+    for (let toResolve of args['load-biblio'] ?? []) {
+      if (/^[./]/.test(toResolve)) {
```

Comment:
```suggestion
      if (/^\.{0,2}\//.test(toResolve)) {
```

Maybe also pull the regexp out since it's used elsewhere.